scripts/spcgeonode/django/initialize.py

```diff
@@ -155,14 +161,15 @@
 
 print("-----------------------------------------------------")
 print("8. Waiting for GeoServer")
+_geoserver_host = os.getenv('GEOSERVER_LOCATION', 'http://geoserver:8080/geoserver')
 for _ in range(60*5):
     try:
-        requests.head("http://geoserver:8080/geoserver")
+        requests.head("{}".format(_geoserver_host))
         break
     except ConnectionError:
         time.sleep(1)
 else:
-    requests.head("http://geoserver:8080/geoserver")
+    requests.head("{}".format(_geoserver_host))
```

Comment:
In Kubernetes deployment sometimes we define different service name to resolve to the pods.
Allow to override this hardcoded value, but keep existing value as default for docker-compose based deployment.